shippable.yml

```diff
@@ -16,7 +16,7 @@ build:
     - cd /home/warp_user/warp/pywarp90 && git pull && make install && make pinstall
 
     #QED LIBRARY
-    - cd /home/warp_user/src/bitbucket.org/berkeleylab/picsar/src/multi_physics
+    - cd /home/warp_user/src/bitbucket.org/berkeleylab/picsar/multi_physics/QED
```

Comment:
We're currently not using shippable right? Should we remove this file altogether from `development`?

Reply:
Not that I am aware of ! However, we might see this as an opportunity to start using LGTM with PICSAR-QED (since I think that LGTM needs this file to work). LGTM should be free for open source software, right? What do you think @NeilZaim  and @ax3l  ?

Reply:
Just closing the loop from slack: I think LGTM is now integrated here.